CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+#### 1.0.0-rc.2
+  * The `@eventHandler` annotation has been replaced with the more general
+    `@reflectable` annotation.
+  * All fields/methods on `JsProxy` objects which you want to expose to JS need
+    to be annotated with `@reflectable`. This results in dramatic code size
+    improvement (from ~1MB to ~200KB for TodoMVC) at relatively minimal cost
```

Comment:
I'd say from 400 to 200, since the other 600 are technically a bug that could be fixed without it